src/coffee/io/reader.coffee

```diff
@@ -6,18 +6,26 @@ iconv = require 'iconv-lite'
 fs = Promise.promisifyAll require('fs')
 Excel = require 'exceljs'
 
+debugLog = _.noop
+
 class Reader
 
   constructor: (@options = {}) ->
+    if @options.debug
+      debugLog = console.log
```

Comment:
Using the debug module, instead of always having code like _READER::options_ you can always create a new class of the debug module, each with it's own scope and use directly.